### PR TITLE
Replace fixture puids with hardcoded puids

### PR DESCRIPTION
**Pull request by ChrisHuynh333**
## What does this PR do and why?
This PR replaces the fixture PUIDs with hardcoded ones so they may be referenced by other fixtures.

## Screenshots or screen recordings
_Screenshots are required for UI changes, and strongly recommended for all other pull requests._

## How to set up and validate locally
1. Ensure all tests pass locally

## PR acceptance checklist
This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.

- [x] I have evaluated the [PR acceptance checklist](https://phac-nml.github.io/irida-next/docs/development/development_processes/code_review#acceptance-checklist) for this PR.

### CODE REVIEW

File: db/schema.rb
Comment (ericenns) [inline db/schema.rb:1]: **nitpick (blocking):** I think this was accidentally committed. Can you remove it and then this pr is good to go.

**Review by ericenns — CHANGES_REQUESTED** [review]
One minor change please

File: db/schema.rb
Comment (ChrisHuynh333) [reply db/schema.rb:1]: Fixed in 81efee6731c3465767433b14011f1f53fdca0a0a

**Review by ericenns — APPROVED** [review]
Looks good to me!

**Pull request merged**